documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+# Breaking Change Policy
```

Comment:
@brendanburns  How to notify customers is a serious problem, and we are soliciting customer feedback .  Having two API states would be ideal, but difficult to implement without architectural changes.  Architectural changes that might support this are planned, but not soon.